server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java

```diff
@@ -371,7 +377,7 @@ public static RealmRepresentation toRepresentation(RealmModel realm, boolean int
         rep.setAttributes(attributes);
 
         if (!internal) {
-            rep = StripSecretsUtils.strip(rep);
+            return StripSecretsUtils.strip(rep);
```

Comment:
Certainly.  I've been splitting up PR's that aren't strictly related (I.E. null values for default procotols).  However, I didn't see the harm in finializing the variable for use with Lambdas since the final output wound up being kind of an if/else.  That function seemed to be imperative anyway in that it was modifying the underlying config values on the list.  Anyway, will un-finalize as the lambda portion is being removed.